src/Commands/QueueWork.php

```diff
@@ -93,8 +93,8 @@ public function run(array $params)
         $maxTime    = $params['max-time'] ?? CLI::getOption('max-time') ?? 0;
         $memory     = $params['memory'] ?? CLI::getOption('memory') ?? 128;
         $priority   = $params['priority'] ?? CLI::getOption('priority') ?? $config->getQueuePriorities($queue) ?? 'default';
-        $tries      = $params['tries'] ?? CLI::getOption('tries');
-        $retryAfter = $params['retry-after'] ?? CLI::getOption('retry-after');
+        $tries      = (int) ($params['tries'] ?? CLI::getOption('tries'));
+        $retryAfter = (int) ($params['retry-after'] ?? CLI::getOption('retry-after'));
```

Comment:
This has to be handled differently. We expect `null` if no value is set.

Reply:
Oh, thanks. Fixed.

Reply:
And should be: 7cafc2d0bb56fe0e2223505a8727211be82351fd

Reply:
Thanks. I made changes to support all the params: https://github.com/codeigniter4/queue/pull/12/commits/440d844007bc3e62fddead9f710a6968b30afbd0